app/controllers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
-  protect_from_forgery with: :exception
+  protect_from_forgery unless: -> { request.format.json? }
```

Comment:
We should only need this protect_from_forgery if we're setting up an API controller, why did you choose to place it here rather than in the ApiController?

Reply:
So I think that we originally had that line in both the api_controller and the application_controller, but then someone told us to remove it from the api_controller. Do you want us to move it to the api_controller?